classes/importer.php

```diff
@@ -253,20 +254,45 @@ public function load_ldap_data_to_table() {
 
             do {
                 if ($ldap_pagedresults) {
-                    ldap_control_paged_result($ldapconnection, $this->config->pagesize, true, $ldap_cookie);
+                    $servercontrols = $this->get_ldap_controls($this->config, $ldap_cookie);
                 }
                 if ($this->config->search_sub) {
                     // Use ldap_search to find first user from subtree.
-                    $ldap_result = ldap_search($ldapconnection, $context, $filter, array('uid', $this->config->user_attribute, 'createtimestamp', 'modifytimestamp'));
+                    $ldap_result = ldap_search(
+                        $ldapconnection,
+                        $context,
+                        $filter,
+                        array('uid', $this->config->user_attribute, 'createtimestamp', 'modifytimestamp'),
+                        0,
```

Comment:
PHP 8 supports named arguments now, so you could shorten this by just `controls: $servercontrols`.
https://wiki.php.net/rfc/named_params

Reply:
good suggestion. handled: 

:heavy_check_mark: